app/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
     package="com.llamalabb.TTT3D">
 
     <application
-        android:name=".tictactoe.TTT3DApplication"
+        android:name=".util.TTT3DApplication"
```

Comment:
The application class should not be in a util package. It is the main booting class of the app.